packages/utils/src/lib/file-system.unit.test.ts

```diff
@@ -57,9 +58,9 @@ describe('crawlFileSystem', () => {
   beforeEach(() => {
     vol.fromJSON(
       {
-        ['README.md']: '# Markdown',
-        ['src/README.md']: '# Markdown',
-        ['src/index.ts']: 'const var = "markdown";',
+        'README.md': '# Markdown',
+        'src/README.md': '# Markdown',
+        'src/index.ts': 'const var = "markdown";',
```

Comment:
Thx!! Is there maybe a lint rule for such cases? I remember I mad the same mistake somewhere else too.

Reply:
Found it! Will add it to shared config:
- https://github.com/code-pushup/eslint-config/issues/26